README.md

```diff
@@ -1,39 +1,102 @@
-# Alpha Site
+# Service Canada Labs / Alpha Site
 
-[![Build Status](https://teamcity.dts-stn.com/app/rest/builds/buildType:(id:Dev_AlphaSite_Build)/statusIcon)](https://teamcity.dts-stn.com/viewType.html?buildTypeId=Dev_AlphaSite_Build&guest=1)
-[![link](https://img.shields.io/badge/link-%F0%9F%94%97-brightgreen)](https://alphasite-main.dev.dts-stn.com/)
-
-
-
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+Service Canada Labs is a product being built by the Digital Experience and Client Data (DECD) unit within Employment and Social Development Canada (ESDC) which is a department of the Government of Canada. This product is part of a larger project occurring within ESDC to modernize benefits delivery to Canadian citizens and residents aptly called Benefits Delivery Modernization (BDM) . The goal of this product is to be a digital platform where the general public is able to be see, test and provide feedback on `Alphas` which are early stage products under development by DECD and partners. This will allow early and rapid collection of user feedback which is vital in delivering useful products with excellent user experience.
 
 ## Getting Started
 
-You can run the development server directly or through docker-compose:
+Run the development server:
 
+**NPM version <6**
 ```bash
 npm i
 npm run dev
-# or
-docker-compose run dev
+```
+
+**NPM version >7**
+```bash
+npm i --legacy-peer-deps
+npm run dev
+```
+
+Create and run a production build locally:
+
+```bash
+npm run build
+npm start
 ```
```

Comment:
Might be a good idea to mention that all the envs need to be set before creating the build, as Next.js is pretty finicky about that.

Reply:
Do you mean as in the build fails if certain ones are not set? I am not sure what those are as I just tested it with a couple client-side and server-side ones and didn't have any failed build.

Reply:
Right, you did specify that the CMS api url is required. I just thought a warning that the build will fail if you don't have the necessary envs would help. I'll approve since you do mention the required env though.

Reply:
@var-kyle done!